src/components/ArticleCard.jsx

```diff
@@ -1,25 +1,31 @@
+import './ArticleCard.css'
 import moment from "moment/moment"
 import { Link } from "react-router-dom"
 import { UserImage } from "./UserImage"
+import { formatCommentCount, formatVoteCount } from '../utils/display'
 
-export const ArticleCard = ({
-    article: { article_id, votes, comment_count, author, created_at, topic, title },
-    authorName
-}) => (<div className="article-card">
-    <div className="card-header">
-        <div className="author-container">
-            <UserImage username={author} className="picture" />
-            <div className="author">{authorName[article_id]}</div>
+export const ArticleCard = ({ article: { article_id, votes, comment_count, author, created_at, topic, title }, authorNameReference }) => (
+    <div className="article-card">
+        <div className="card-header">
+            <div className="card-title">{title}</div>
+            <div className="card-votes">{`${formatVoteCount(votes)}`}</div>
         </div>
-        <div className="votes">{`${votes} vote${votes === 1 ? '' : 's'}`}</div>
-    </div>
-    <div className="card-title">{title}</div>
-    <Link to={`/articles/${article_id}`}>
-        <button className="read-article-button">Read Article</button>
-    </Link>
-    <div className="card-footer">
-        <div className="created-at">{moment(created_at).format('M/D/YYYY')}</div>
-        <div>{`${comment_count} comment${comment_count === 1 ? '' : 's'}`}</div>
-        <div>{topic}</div>
-    </div>
-</div>)
\ No newline at end of file
+        <div className="card-mid">
+            <div className="author-container">
+                <UserImage username={author} className="card-picture" />
+                <div className="author">{authorNameReference[article_id]}</div>
```

Comment:
It is personal preference, but you are exposing the real names of the user with explicit permission - was there a particular reason as to why you did this?

Reply:
I figured since it's a news site, the author would use their name to publish. I can take all names out on the site. Is there anywhere I should use the names at all in this project?